sync-quickstart-android/build.gradle

```diff
@@ -48,19 +47,6 @@ dependencies {
     implementation 'com.twilio:sync-android:0.8.6'
     implementation 'com.koushikdutta.ion:ion:3.0.8'
     implementation 'com.jakewharton.timber:timber:4.3.0'
-
-    implementation 'com.crashlytics.sdk.android:crashlytics:2.10.1'
```

Comment:
we need crashlytics, why do you remove it?

Reply:
Probably we should update it instead of removing. We even have a ticket about that: RTDSDK-2145 Feel free to take it.